InventoryInStorePickup/Model/SearchRequest/Area.php

```diff
@@ -22,42 +22,18 @@ class Area implements AreaInterface
     /**
      * @var string
      */
-    private $country;
-
-    /**
-     * @var string|null
-     */
-    private $region;
-
-    /**
-     * @var string|null
-     */
-    private $city;
-
-    /**
-     * @var string|null
-     */
-    private $postcode;
+    private $searchTerm;
 
     /**
      * @param int $radius
-     * @param string $country
-     * @param string|null $region
-     * @param string|null $city
-     * @param string|null $postcode
+     * @param string $searchTerm
      */
     public function __construct(
         int $radius,
```

Comment:
According to the `AreaBuilder`, any of the parameters can be empty, - so we will have an Exception in this case.

Either validation or nullable type hints should be provided.

Reply:
Could you share a link?